services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java

```diff
@@ -92,6 +93,14 @@ void installDropFlowCustom(final DatapathId dpid, String dstMac, String dstMask,
                                final long cookie, final int priority) throws SwitchOperationException;
 
 
+    /**
+     * Installs the rule that drops packets sent from this particular switch (in order to prevent creating self-looped
+     * ISLs).
+     * @param dpid datapathId of switch
+     * @throws SwitchOperationException in case of errors
+     */
+    void installDropLoopRule(DatapathId dpid) throws SwitchOperationException;
```

Comment:
You don't need it to be public. At this moment only SwitchManager is responsible for all "default"(terrible term) rules... External word can(and should not) control over setup/remove of this rules.